core/src/main/java/org/apache/iceberg/SerializableTable.java

```diff
@@ -278,6 +278,10 @@ public TableScan newScan() {
     return lazyTable().newScan();
   }
 
+  public IncrementalAppendScan newIncrementalAppendScan() {
```

Comment:
can you also override `newIncrementalChangelogScan()` here please?

Reply:
thanks @nastra for the review! added missing `@Override` annotation

Reply:
+ newIncrementalChangelogScan() override